resources/dicts/patrols/general/medcat.json

```diff
@@ -3571,5 +3571,250 @@
+	"decline_text": "Noticing storm clouds above, p_l decides to head back to camp before it starts raining. Their catch-up would have to wait.",
+        "chance_of_success": 60,
+        "success_outcomes": [
+            {
+                "text": "Noticing a mullein patch, p_l begins to excitedly explain how useful a find it was, before pausing - s_c has no idea what {PRONOUN/p_l/subject}{VERB/p_l/'re/'s} talking about, {VERB/s_c/do/does} {PRONOUN/s_c/subject}? s_c admits that, no, not entirely, but {PRONOUN/s_c/subject}{VERB/s_c/'re/'s} more than happy for p_l to explain it to {PRONOUN/s_c/object}! By the end of the patrol, not only is s_c a mullein-master, but {PRONOUN/s_c/object} and p_l's bond is renewed.",
+                "exp": 20,
+                "weight": 20,
+		"stat_trait": ["calm", "charismatic", "compassionate", "faithful", "insecure", "loving", "loyal", "nervous", "responsible", "righteous", "sincere", "thoughtful", "wise", "careful"],
+		"can_have_stat": ["r_c"],
+                "herbs": ["mullein"],
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["platonic", "comfort", "trust"],
+                        "amount": 5
+                    }
+                ],
+	        "art": "gen_med_collectingmullein"
+            },
+            {
+                "text": "s_c purrs, brushing {PRONOUN/s_c/poss} pelt against p_l's - s_c may not be the most talkative cat, but {PRONOUN/s_c/subject} {VERB/s_c/want/wants} p_l to know that {PRONOUN/s_c/subject} {VERB/s_c/appreciate/appreciates} p_l's presence. p_l purrs back, draping {PRONOUN/p_l/poss} tail across {PRONOUN/p_l/poss} parent's shoulders and continuing the patrol in a comfortable silence.",
+                "exp": 20,
+                "weight": 20,
+		"stat_trait": ["calm", "gloomy", "grumpy", "strange", "lonesome"],
+		"can_have_stat": ["r_c"],
+                "herbs": ["random_herbs"],
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["platonic", "comfort", "trust"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "Though s_c may not always understand the path of a medicine cat, {PRONOUN/s_c/subject} wants p_l to know that {PRONOUN/s_c/subject}{VERB/s_c/'re/'s} proud of {PRONOUN/p_l/object} anyway, and is glad p_l is happy in their role. Surprised - but appreciative - of the reassurance, p_l smiles and promises s_c that just because they walk different paths doesn't mean {PRONOUN/p_l/subject}'ll stop needing s_c.",
```

Comment:
```suggestion
                "text": "Though s_c may not always understand the path of a medicine cat, {PRONOUN/s_c/subject} wants p_l to know that {PRONOUN/s_c/subject}{VERB/s_c/'re/'s} proud of {PRONOUN/p_l/object} anyway, and is glad p_l is happy in {PRONOUN/p_l/poss} role. Surprised - but appreciative - of the reassurance, p_l smiles and promises s_c that just because they walk different paths doesn't mean {PRONOUN/p_l/subject}'ll stop needing s_c.",
```